datasets/craigslist_bargains/README.md

```diff
@@ -0,0 +1,263 @@
+  - `Price`: Price(s) of the item. Default value for missing: (`-1`)
+  - `Title`: Title(s) of the item
+- `utterance`: Utterance for each turn in the dialogue, corresponding to the agent in `agent_turns`. The utterance may be an empty string (`''`) for some turns if multiple dialogue acts take place after an utterance (e.g. there are often multiple dialogue acts associated with the closing of the bargaining process after all utterances have completed to describe the conclusion of the bargaining).
+
+### Data Splits
+
+This dataset contains three splits, `train`, `validation` and `test`. Note that `test` is not provided with `dialogue_acts` information as described above. To ensure schema consistency across dataset splits, the `dialogue_acts` field in the `test` split is populated with the default values: `{"price": -1.0, "intent": ""}`
+
+The counts of examples in each split are as follows:
+
+|                            | Train  | Valid | Test |
+| Input Examples             | 5247   | 597   | 838  |
+| Average Dialogue Length    | 9.14   | 9.17  | 9.24 |
+
+Note that 
+
+## Dataset Creation
+
+From the [source paper](https://arxiv.org/pdf/1808.09637.pdf) for this dataset: 
+
+> To generate the negotiation scenarios, we
+> scraped postings on sfbay.craigslist.org
+> from the 6 most popular categories (housing, furniture, cars, bikes, phones, and electronics). Each
+> posting produces three scenarios with the buyer’s
+> target prices at 0.5x, 0.7x and 0.9x of the listing
+> price. Statistics of the scenarios are shown in Table 2.
+> We collected 6682 human-human dialogues on
+> AMT using the interface shown in Appendix A
+> Figure 2. The dataset statistics in Table 3 show
+> that CRAIGSLISTBARGAIN has longer dialogues
+> and more diverse utterances compared to prior
+> datasets. Furthermore, workers were encouraged
+> to embellish the item and negotiate side offers
+> such as free delivery or pick-up. This highly relatable scenario leads to richer dialogues such as
+> the one shown in Table 1. We also observed various persuasion techniques listed in Table 4 such as
+> embellishment,
+
+### Curation Rationale
+
+See **Dataset Creation**
+
+### Source Data
+
+See **Dataset Creation**
+
+#### Initial Data Collection and Normalization
+
+See **Dataset Creation**
+
+#### Who are the source language producers?
+
+See **Dataset Creation**
+
+### Annotations
+
+If the dataset contains annotations which are not part of the initial data collection, describe them in the following paragraphs.
+
+#### Annotation process
+
+Annotations for the `dialogue_acts` in `train` and `test` were generated via a rules-based system which can be found in [this script](https://github.com/stanfordnlp/cocoa/blob/master/craigslistbargain/parse_dialogue.py)
+
+#### Who are the annotators?
+
+### Personal and Sensitive Information
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+### Discussion of Biases
+
+### Other Known Limitations
+
```

Comment:
can you add `[More Information Needed]` instead of leaving the sections empty please ?

Reply:
Done!